backend/src/models/Report.ts

```diff
@@ -0,0 +1,37 @@
+import { Table, Column, Model, DataType, ForeignKey, BelongsTo } from 'sequelize-typescript';
+import { Candidate } from './Candidate';
+
+@Table
+export class Report extends Model {
+  @Column({
+    type: DataType.ENUM,
+    values: ['clear', 'consider', 'schedule']
```

Comment:
nit: may be you should create actual typescript enum for this. It will reduce hard coding and repeatation.